apps/prairielearn/src/server.js

```diff
@@ -729,6 +699,8 @@ module.exports.initExpress = function () {
   // files.
   // TODO: if we can determine that these routes are no longer receiving
   // traffic in the future, we can delete these.
+  //
+  // TODO: the only internal usage of this is in the `pl-drawing` element. Fix that.
```

Comment:
👀 I think we need to start providing something like `client_files_element_url` to individual elements.

Reply:
Makes sense!